lib/project_types/script/config/extension_points.yml

```diff
@@ -16,6 +19,9 @@ shipping_methods:
     typescript:
       package: "@shopify/scripts-checkout-apis"
       repo: "https://github.com/Shopify/scripts-apis-examples"
+    wasm:
```

Comment:
Also, what do you think about adding a CI check later to ensure every API has a wasm language?

Reply:
I can definitely add it for every API.
Previously, in order to avoid adding Wasm to every API in the file, I was injecting it elsewhere.
A CI check is probably necessary now that we're using sparse checkout during create.

Reply:
https://github.com/Shopify/script-service/issues/4263